src/Commands/Remote/SSHBaseCommand.php

```diff
@@ -252,7 +267,11 @@ private function getOutputCallback()
      */
     private function getCommandLine($command_args)
     {
-        array_unshift($command_args, $this->command);
+        // Retrieve the trace ID from the TraceId class
+        $trace_id = TraceId::getTraceId();
+
+        // Prepend the trace ID to the command arguments
+        array_unshift($command_args, "TRACE_ID={$trace_id}", $this->command);
```

Comment:
Have you tried this? I think that escapeArgument is going to wrap TRACE_ID=uuid in quotes, because "=" isn't an allowed character, and I think that wrapping it in quotes will produce the wrong results. You could try fixing the code sniff and see if the tests pass, but I recommend handling the env vars separately from the command args.